docs/source/developer_guide/guides/3_simple_cpp_stage.md

```diff
@@ -89,6 +89,7 @@ While our Python implementation accepts messages of any type (in the form of Pyt
 To start with, we have our Morpheus and MRC-specific includes:
 
 ```cpp
+#include "morpheus/export.h"
```

Comment:
We should use angle brackets to be consistent in the developer guide includes

Reply:
Converted!